--- old/sonara/android/app/src/main/java/com/castmobile/sender/ui/AppComponents.kt	2026-09-29 11:26:01.000000000 +0000
+++ new/sonara/android/app/src/main/java/com/castmobile/sender/ui/AppComponents.kt	2026-10-05 11:17:19.559881372 +0000
@@ -1029,8 +1029,7 @@
             val tabs = listOf(
                 DockTab(0, Icons.Outlined.Home, Icons.Filled.Home, "Home"),
                 DockTab(1, Icons.Outlined.LibraryMusic, Icons.Filled.LibraryMusic, "Library"),
-                DockTab(2, Icons.Outlined.MusicNote, Icons.Filled.MusicNote, "All Music"),
-                DockTab(3, Icons.Outlined.Palette, Icons.Filled.Palette, "Lab")
+                DockTab(2, Icons.Outlined.Palette, Icons.Filled.Palette, "Lab")
             )
 
             for (tab in tabs) {
